pages/partials/instructorGradingPanel.ejs

```diff
@@ -0,0 +1,37 @@
+<% if (authz_data.has_instructor_view || authz_data.has_course_permission_view) { %>
+    <div class="card mb-4 border-success">
+      <div class="card-header bg-success">Grading Panel</div>
+      <div class="card-body">
+        <% if (typeof submission_updated !== 'undefined' && submission_updated === true) { %>
+            <blockquote style="color: red">Submission updated</blockquote>
+        <% } %>
+        <form name="instance_question-manual-grade-update-form" method="POST">
+            <input type="hidden" name="__action" value="add_manual_grade">
+            <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+            <input type="hidden" name="assessment_instance" value="<%= assessment_instance.assessment_id %>">
```

Comment:
Is this the assessment_instance_id (in which case the value is wrong) or the assessment_id (in which case the name is wrong)?

Reply:
Dropped the `_id` suffix. Thanks for the catch.